test_assets/features/support/world.js

```diff
@@ -1,11 +1,10 @@
 module.exports = function() {
-  this.World = function(callback) {
-    var world = {};
+  this.World = function() {
```

Comment:
Is the change to using this as the World rather than initialising a new one, a stylistic change or is needed for something else?